theano/compile/mode.py

```diff
@@ -199,7 +199,7 @@ def apply(self, fgraph):
 
 # rearranges elemwise expressions
 optdb.register('canonicalize', gof.EquilibriumDB(ignore_newtrees=False),
-               1, 'fast_run', 'fast_compile')
+               1, 'fast_run', 'fast_compile', 'canonicalize_db')
```

Comment:
We named it `_db` because it includes only the `gof.DB` itself, not all the optimizations that usually go in it.
I'm not sure what you have in mind. If you want to clarify that it is an EquilibriumDB, then we could maybe tag it `canonicalize_eqdb`. But the point is that nothing else than this empty DB should use that tag.